doc/src/Examples.dox

```diff
@@ -48,47 +46,58 @@
- * # the heart of all our interactions with a the API. It allows
- * # us to query the available managers, and pick one to talk to.
- * session = Session(host_interface, logger, factory)
+ * # We can now create an OpenAssetIO ManagerFactory. The ManagerFactory
+ * # allows us to query the available managers, and pick one to talk to.
+ * managerFactory = ManagerFactory(host_interface, factory_impl, logger)
  * @endcode
  *
  * @section examples_picking_a_manager Setting up a Manager
  *
- * This example makes use of the newly initialized session to show how to
- * configure a @ref session to use a specific manager (it assumes that
- * some example @ref asset_management_system has a plugin, installed on
- * @ref plugin_path_var).
+ * This example makes use of the newly initialized factory to show how
+ * to construct and configure a specific manager (it assumes
+ * that some example @ref asset_management_system has a plugin,
+ * installed on @ref plugin_path_var).
  *
  * We will be providing an example manager implementation soon!
  *
  * @code{.py}
- * managers = session.registeredManagers()
+ * availableManagers = managerFactory.availableManagers()
  * > {
- * >    'org.openassetio.example.manager': {
- * >         'name': 'Example Asset Manager',
- * >         'identifier': 'org.openassetio.example.manager',
- * >         'info': {},
- * >         'plugin': <class '5709f502e48331b76857b0f7274cbcf5.ExampleManagerPlugin'>
+ * >    'org.openassetio.example.manager':
+ * >         ManagerFactory.ManagerDetail(
+ * >             identifier='org.openassetio.example.manager',
+ * >             displayName='Example Asset Manager',
+ * >             info={},
+ * >             settings={"server": ""})
  * >    }
  * > }
  *
- * # We then tell the session which manager we'd like to use.
- * # Each session can only communicate with one manager at once.
- * # If we wanted to bridge between two, we'd create two sessions.
- * session.useManager('org.openassetio.example.manager')
- *
- * # At this point the manager hasn't yet been initialized. This is
- * # done lazily when it's first needed. This might seem odd here, but
- * # it makes life easier in practice in real-world integrations as often
- * # numerous call sites may wish to obtain access to the manager.
- * manager = session.currentManager()
+ * # Once we know which manager we wish to use, we can ask the factory
+ * # to create one for us.
+ * manager = managerFactory.createManager('org.openassetio.example.manager')
+ *
+ * # We now have an instance of the requested manager, but it is not
+ * # quite ready for use yet. The manager returned by the
+ * # ManagerFactory needs to be initialized before it can be used to
+ * # query or publish assets. Setup is split into two stages to allow
+ * # adjustments to its settings to be made prior to use if required.
+ *
+ * # A manager's current (or in this case default) settings can be
+ * # queried if needed:
+ * settings = manager.settings()
```

Comment:
[question] I like your rewordings :+1: . 

I guess it makes more sense to get the settings from the manager instance once created, rather than extracting from the `availableManagers`.  

But then that begs the question, is there really any need for `settings` (for that matter, `info`) in the `availableManagers` output?

Reply:
Good Q - looks like we added it in #522 - it wasn't in the current python implementation. Shall we bin if off?

Reply:
Removed from Examples.dox in this PR in e9cefa2a and removed from the codebase in #522 (https://github.com/OpenAssetIO/OpenAssetIO/pull/522#issuecomment-1192444052)

Reply:
@feltech when you do the squashy squash, we'll need to remove the inclusion of the settings in the output in the example.

Reply:
Haha, sorry, great minds think alike - sorry I should have known you'd be on it already.